src/components/sound.js

```diff
@@ -24,44 +23,33 @@ module.exports.Component = registerComponent('sound', {
 
   update: function (oldData) {
     var data = this.data;
-    var diffData = diff(oldData || {}, data);
     var el = this.el;
     var sound = this.sound;
-    var src = data.src;
-    var srcChanged = 'src' in diffData;
+    var srcChanged;
+
+    oldData = oldData || {};
```

Comment:
Yeah, I think so. Previously, it was `null` so that we could determine whether it was the first update. But that could be done instead by just setting a flag from `init`. Having to check for `null` gets in the way more than it is useful.

Separate PR? Or this one.